index.js

```diff
@@ -24,7 +33,7 @@ class MessageHider extends Plugin {
 						document.getElementById(
 							`chat-messages-${message.id}`
 						).style.display = 'none';
-
+						if (powercord.pluginManager.get("powercord-message-hider").settings.get('popup', 'true') == false ) { return }
```

Comment:
Will not work, you're checking if the popup is enabled after the message is hidden.